index.html

```diff
@@ -395,30 +409,34 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
             <fieldset class="field">
               <label class="label" for="username">Username</label>
               <div class="control has-icons-left has-icons-right">
-                <input id="username" class="input is-success" type="text" name="username">
-                <span class="icon is-small is-left">
+                <input id="username" class="input is-success" type="text" name="username"
+                  placeholder="Enter your username" aria-describedby="usernameHelp"
+                  aria-invalid="false">
+                <span class="icon is-small is-left" aria-hidden="true">
                   <i class="fas fa-user"></i>
                 </span>
-                <span class="icon is-small is-right">
+                <span class="icon is-small is-right" aria-hidden="true">
                   <i class="fas fa-check"></i>
                 </span>
               </div>
-              <p class="help is-success">This username is available</p>
+              <p id="usernameHelp" role="alert" class="help is-success">This username is
+                available</p>
```

Comment:
Also this is a good case to use `aria-describedby` attr
Check example here: https://www.tpgi.com/using-aria-describedby-to-provide-helpful-form-hints/

Reply:
<img width="565" alt="Screenshot 2024-02-16 at 20 48 40" src="https://github.com/yurassic/accessibility-training/assets/10263616/08cc445e-ead4-4994-ae26-d0ba26fe421c">
But it was added to input